src/App.tsx

```diff
@@ -1,24 +1,75 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, {
+  useCallback,
+  useContext,
+  useEffect,
+  useState,
+} from 'react';
 
-const USER_ID = 0;
+import { getTodos } from './api/todos';
+import { UserWarning } from './UserWarning';
+import { TodoList } from './components/TodoList';
+import { ErrorMessage } from './components/ErrorMessage';
+import { Loader } from './components/Loader';
+import { AppContext } from './components/AppContext';
+import { Header } from './components/Header';
+import { Footer } from './components/Footer';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
+  const [isLoadingTodos, setIsLoadingTodos] = useState(false);
+
+  // eslint-disable-next-line no-unused-vars
```

Comment:
??

Reply:
If I don't use it, it doesn't allow me to deploy the project. There is written that I imported useContext, but I don't use it (I do use it). The mentor's advice was to disable eslint rule for this line of code